crates/noirc_evaluator/src/ssa_refactor/ir/instruction.rs

```diff
@@ -126,12 +126,16 @@ impl Instruction {
 pub(crate) enum TerminatorInstruction {
     /// Control flow
     ///
-    /// Jump If
+    /// Branch If
     ///
-    /// Jumps to the specified `destination` with
-    /// arguments, if the condition
-    /// if the condition is true.
-    JmpIf { condition: ValueId, destination: BasicBlockId, arguments: BlockArguments },
+    /// Branches to the specified `then_destination` with `arguments` if the condition is true,
+    /// otherwise branches to the `else_destination` with `arguments`.
```

Comment:
```suggestion
     /// If the condition is true: jump to the specified `then_destination` with `arguments`.
     /// Otherwise, jump to the specified `else_destination` with `arguments`.
```